docs/source/install.rst

```diff
@@ -295,17 +46,17 @@ Troubleshooting
 Failure to Install Python Package
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-If your attempt to install Python packages such as NumPy fails, please upgrade your pip install as follows
+If your attempt to install Python packages such as NumPy fails, please upgrade your pip install as follows:
 
-    On macOS and Linux::
+    On OS X and Linux::
```

Comment:
All references to 'OS X' should instead be changed to 'macOS'. This line used to show 'macOS'.